website/docs/r/vm.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_vm"
+sidebar_current: "docs-vcd-resource-vm"
+description: |-
+  Provides a vCloud Director VM resource. This can be used to create, modify, and delete standalone VMs.
+---
+
+# vcd\_vm
+
+Provides a VMware Cloud Director standalone VM resource. This can be used to create, modify, and delete standalone VMs.
```

Comment:
For a novice, it's very easy to miss the "standalone" part and think it's any VM. Let's highlight somehow that this is specifically for standalone. We can upper case it or make it bold or italic.

```suggestion
Provides a VMware Cloud Director standalone VM resource. This can be used to create, modify, and delete Standalone VMs.
```

Reply:
changed